assets/src/edit-story/components/canvas/canvasProvider.js

```diff
@@ -51,9 +51,17 @@ function CanvasProvider({ children }) {
   } = useEditingElement();
 
   const {
-    state: { currentPage, selectedElementIds },
-    actions: { toggleElementInSelection, setSelectedElementsById },
-  } = useStory();
+    currentPage,
+    selectedElementIds,
+    toggleElementInSelection,
+    setSelectedElementsById,
+  } = useStory(({ // eslint-disable-next-line no-shadow
+    state: { currentPage, selectedElementIds }, actions: { toggleElementInSelection, setSelectedElementsById } }) => ({
```

Comment:
Not sure why the linter prefers this syntax when the return value object is wrapped.